### PR TITLE
displays if filters are selected in sidebar

### PR DESCRIPTION
**Pull request by Danwhy**
ref #263
Displays the name of the tag if only one is selected, or the number of tags if multiple are selected

### CODE REVIEW

File: web/static/js/sidebar.js [hunk 1]
```diff
@@ -67,3 +78,34 @@ function toggleArrows(type) {
   toggleClasses(select(".select-" + type + "-filters > h5 > .fa-up"), ["filter-arrow-hide"]);
   toggleClasses(select(".select-" + type + "-filters > h5 > .fa-down"), ["filter-arrow-hide"]);
  }
+
+function updateSelected(form) {
+  var formData = new FormData(form);
+
+  for (var value of formData.entries()) {
```
Comment (shouston3) [inline web/static/js/sidebar.js]: Not too sure about the browser compatibility of `for... of` loops. Couldn't find info on MDN or CanIUse, but [here](https://docs.microsoft.com/en-us/scripting/javascript/reference/for-dot-dot-dot-of-statement-javascript) it says `Not supported in the following document modes... Internet Explorer 8 standards, Internet Explorer 9 standards, Internet Explorer 10 standards`

I'm a little unclear how supported this is tbh. You could probably write something similar with a `for` loop or `for...in` loop or something.

**Review by samhstn — CHANGES_REQUESTED** [review]
Not to sure about the above btw, could you clarify.

Otherwise, looks great

**Review by samhstn — APPROVED** [review]
Looks great

**Review by nelsonic — APPROVED** [review]
@Danwhy great work. 🎉

**Review by SimonLab — APPROVED** [review]
Looks good :+1: , elm :wink: ?

**Pull request merged**